sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -487,3 +490,39 @@ def __init__(self):
         self.signed_service = None
         self.signed_version = None
         self.value = None
+
+class TokenAudience(object):
```

Comment:
I don't think you need to inherit from `object`. I think that is an old Python 2 thing? Maybe look it up since I'm not sure.